osu.Framework.Tests/Visual/UserInterface/TestSceneMarkdownContainer.cs

```diff
@@ -301,6 +301,12 @@ public void TestAutoLinkInline()
             AddAssert("has correct autolink", () => markdownContainer.AutoLinks[0].Url == "https://discord.gg/ppy");
         }
 
+        [Test]
+        public void TestEmptyFencedBlock()
+        {
+            AddStep("set empty fenced block", () => markdownContainer.Text = @"```");
```

Comment:
Is this intentionally unbalanced? Seems to still repro with a balanced markdown string (which sits better with me for this test):

```suggestion
            AddStep("set empty fenced block", () => markdownContainer.Text = @"``````");
```

Reply:
Intentionally unbalanced just to show that even this unbalancedness draws a background like in other services.

Reply:
Hmm, I'd probably rename the method to `TestUnbalancedFencedBlock` and maybe add a second one with your original name and my proposal.